README.md

```diff
@@ -33,7 +33,7 @@ Using [Oh-my-zsh](https://github.com/robbyrussell/oh-my-zsh):
 
 2. Activate the plugin in `~/.zshrc`:
 
-        plugins=( [plugins...] history-substring-search)
+        plugins=( [plugins...] zsh-history-substring-search)
```

Comment:
It works fine for me without the prefix, and doesn't work with it.  I think it's correct the way it is